### ISSUE TITLE
"Tidy up channel" operation stalls with no visible progress

### ISSUE THREAD

**Issue by starpotion** [axcore/tartube#423]
### What operating system are you using?
Windows 11

### What version of Tartube are you using?
v2.4.0

### What happens when you run Tartube from a terminal window?
```
rekyuu@REKYUU-PC MINGW64 ~
$ cd /home/user/tartube

rekyuu@REKYUU-PC MINGW64 /home/user/tartube
$ python3 tartube/tartube

(tartube:21476): GLib-GIO-WARNING **: 17:30:34.977: Unexpectedly, UWP app `GravitGmbH.GravitDesigner_3.5.73.0_x64__78ng2gh36rjtw' (AUMId `GravitGmbH.GravitDesigner_78ng2gh36rjtw!GravitDesigner') supports 1 extensions but has no verbs
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'strikethrough-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'underline-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'weight-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gio.py:42: Warning: unable to set property 'style-set' of type 'gboolean' from value of type 'gchararray'
  return Gio.Application.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gtk.py:585: Warning: unable to set property 'strikethrough-set' of type 'gboolean' from value of type 'gchararray'
  return Gtk.Dialog.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gtk.py:585: Warning: unable to set property 'underline-set' of type 'gboolean' from value of type 'gchararray'
  return Gtk.Dialog.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gtk.py:585: Warning: unable to set property 'weight-set' of type 'gboolean' from value of type 'gchararray'
  return Gtk.Dialog.run(self, *args, **kwargs)
C:/Users/rekyuu/AppData/Local/Tartube/msys64/mingw64/lib/python3.9/site-packages/gi/overrides/Gtk.py:585: Warning: unable to set property 'style-set' of type 'gboolean' from value of type 'gchararray'
  return Gtk.Dialog.run(self, *args, **kwargs)
Traceback (most recent call last):
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/dialogue.py", line 469, in on_clicked
    method(response_dict['data'])
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/mainapp.py", line 11847, in tidy_manager_start
    self.current_manager_obj = tidy.TidyManager(self, choices_dict)
  File "C:/Users\rekyuu\AppData\Local\Tartube\msys64\home\user\tartube\tartube/tidy.py", line 166, in __init__
    self.remove_no_url_flag = choices_dict['remove_no_url_flag']
KeyError: 'remove_no_url_flag'
```
![image](https://user-images.githubusercontent.com/10054020/167278128-2d94506d-242e-4c7d-9b06-d84ad8fc98f7.png)

**Comment by axcore** [comment]
I cannot reproduce this error. 

After examining the code, the only explanation I can think of is that your source code has been corrupted. So please first try re-installing Tartube, first removing the old install.

**Comment by tsk-2222** [comment]
It happens to me also (Windows 10, Tartube v2.4.093):
...snip...
```
D:/PortableApps/tartube/mingw64/lib/python3.10/site-packages/gi/overrides/Gtk.py:585: Warning: unable to set property 'style-set' of type 'gboolean' from valof type 'gchararray'
  return Gtk.Dialog.run(self, *args, **kwargs)
Traceback (most recent call last):
  File "D:/PortableApps\tartube\home\user\tartube\tartube/mainwin.py", line 16716, in on_video_index_tidy
    self.app_obj.tidy_manager_start(choices_dict)
  File "D:/PortableApps\tartube\home\user\tartube\tartube/mainapp.py", line 12094, in tidy_manager_start
    self.current_manager_obj = tidy.TidyManager(self, choices_dict)
  File "D:/PortableApps\tartube\home\user\tartube\tartube/tidy.py", line 166, in __init__
    self.remove_no_url_flag = choices_dict['remove_no_url_flag']
KeyError: 'remove_no_url_flag'
```

**Comment by axcore** [comment]
Should be fixed in v2.4.149 (only available from github/pypi at the moment, but it will be in the next full release. Installations on MS Windows can be updated manually, see [here](https://github.com/axcore/tartube#71-tartube-wont-installwont-rundoesnt-work).)
